packages/core/src/TestComponents.stories.tsx

```diff
@@ -0,0 +1,458 @@
+import { Test as DropdownTestStory } from "./Dropdown/stories/Dropdown.stories";
+import { Main as DropDownMenuMainStory } from "./DropDownMenu/DropDownMenu.stories";
+import {
+  Minimal as EmptyStateMinimalStory,
+  WithAction as EmptyStateWithActionStory,
+} from "./EmptyState/EmptyState.stories";
+import { TheDesignSystemColumns as GridTheDesignSystemColumnsStory } from "./Grid/Grid.stories";
+import { Variants as IconButtonVariantsStory } from "./IconButton/IconButton.stories";
+import { Test as InlineEditorTestStory } from "./InlineEditor/InlineEditor.stories";
+import { Variants as InputVariantsStory } from "./Input/Input.stories";
+import { Main as KpiMainStory } from "./Kpi/Kpi.stories";
+import { WithIcons as ListContainerWithIconsStory } from "./ListContainer/ListContainer.stories";
+import { Variants as LoadingVariantsStory } from "./Loading/Loading.stories";
+import { Test as MultiButtonTestStory } from "./MultiButton/stories/MultiButton.stories";
+import { Main as OverflowTooltipMainStory } from "./OverflowTooltip/OverflowTooltip.stories";
+import { Main as PaginationMainStory } from "./Pagination/Pagination.stories";
+import { Main as PanelMainStory } from "./Panel/Panel.stories";
+import { Determinate as ProgressBarDeterminateStory } from "./ProgressBar/ProgressBar.stories";
+import { Test as RadioTestStory } from "./Radio/Radio.stories";
+import {
+  Horizontal as RadioGroupHorizontalStory,
+  Variants as RadioGroupVariantsStory,
+} from "./RadioGroup/RadioGroup.stories";
+import { Main as ScrollToHorizontalMainStory } from "./ScrollTo/Horizontal/ScrollToHorizontal.stories";
+import { Main as ScrollToVerticalMainStory } from "./ScrollTo/Vertical/ScrollToVertical.stories";
+import { Test as SelectTestStory } from "./Select/Select.stories";
+import { Variants as SelectionListVariantsStory } from "./SelectionList/SelectionList.stories";
+import { Main as SimpleGridMainStory } from "./SimpleGrid/SimpleGrid.stories";
+import { Variants as SkeletonVariantsStory } from "./Skeleton/Skeleton.stories";
+import {
+  RangeVariants as SliderRangeVariantsStory,
+  Variants as SliderVariantsStory,
+} from "./Slider/Slider.stories";
+import { Variants as SnackbarVariantsStory } from "./Snackbar/Snackbar.stories";
+import { Test as StackTestStory } from "./Stack/Stack.stories";
+import { Variants as SwitchVariantsStory } from "./Switch/Switch.stories";
+import { Test as TabsTestStory } from "./Tabs/Tabs.stories";
+import { Test as TagTestStory } from "./Tag/stories/Tag.stories";
+import { Variants as TagsInputVariantsStory } from "./TagsInput/TagsInput.stories";
+import { Variants as TextAreaVariantsStory } from "./TextArea/TextArea.stories";
+import { Multiple as ToggleButtonMultipleStory } from "./ToggleButton/ToggleButton.stories";
+import { Test as TypographyTestStory } from "./Typography/Typography.stories";
+import { setupChromatic } from ".storybook/setupChromatic";
+
+const meta: Meta = {
+  title: "Components/Test",
```

Comment:
Maybe we should bring back the `Tests/*` directory under `docs/`. These tests feel a bit unsorted 🤔 

What do you think?

Reply:
Agreed, I'll do that